index.spec.js

```diff
@@ -30,6 +32,15 @@ test('should render three button separated by a custom separator', () => {
 	expect(actual.text()).toEqual('One:Two:Three');
 });
 
+test('should render three button separated by react element <br/>', () => {
+	const actual = shallow(
+		createElement(Group, { separator: React.createElement('br') }, elements)
+	);
+
+	expect(console.error).not.toBeCalled();
```

Comment:
(Sorry for bothering you again.) It’s probably better to override `console.error` inside the test and restore in the end of it. Otherwise this test would break in case of any `console.error` in the app.

Like this:
https://github.com/sapegin/mrm-core/blob/c03fa611855150c75028bee5d649075ffa6d5d79/src/__tests__/core.spec.js#L49-L57

But I don’t know the best way of doing that ;-|